pywbemcli/_common.py

```diff
@@ -282,12 +282,14 @@ def create_cimvalue(cim_type, value_str, is_array):
     CIMValue or list of CIMValue elements.
 
     Parameters:
-      CIM_Type: TODO
-        CIMType for this value.
+      CIM_Type: (:term: `string`)
+        CIMType for this value. The CIM data type name for the CIM object.
+        See :ref:`CIM data types` for valid type names.
 
       Value_str (:term: `string`):
```

Comment:
Change "Value_str" in the description to "value_str" to match the lexical case of the parameter.